src/components/select/select.component.ts

```diff
@@ -325,7 +325,7 @@ export class SelectComponent<T> implements OnInit, OnChanges, OnDestroy, Control
         }
 
         // when the user types and the value is not empty then we should open the dropdown except for non printable keys.
-        if (event.ctrlKey === false && event.shiftKey === false && event.altKey === false && event.metaKey === false && !(event.keyCode === TAB) && !(event.keyCode === ESCAPE)) {
+        if (event.key.length <= 1) {

```

Comment:
Is there a case where we would want to open the dropdown etc, when the length is less than 1?

Reply:
None that I can think of. The drop down will open from pressing the Enter key and arrow keys. These are handled in a separate event.

Reply:
Ok, then could we change this to be `event.key.length === 1` instead? Also the branch is out of date from eh base branch which needs to be resolved:

![image](https://user-images.githubusercontent.com/20795331/72725558-f6747700-3b7d-11ea-923f-f5052daec082.png)

Reply:
sure thing

Reply:
Rebased